packages/web/components/cards/hero-card.tsx

```diff
@@ -45,13 +45,15 @@ export const HeroCard: React.FunctionComponent<{
       </div>
       <div
         onClick={handleAppClicked}
-        className="heroImage relative flex h-[400px] cursor-pointer items-end overflow-hidden rounded-lg"
+        className="heroImage relative flex h-[400px]  cursor-pointer items-end overflow-hidden rounded-lg sm:h-[300px]"
       >
-        <img
-          className="backgroundImage absolute top-0 left-0 z-10 h-full w-full bg-cover bg-center bg-no-repeat"
-          src={imageUrl}
-          alt="hero image"
-        />
+        <div
```

Comment:
why change `img` to `div`?

Reply:
The image behaved strangengly when it comes to automatic resizing via Tailwind and we had it twice (background and actual image <img>) which caused a double image effect for thumbnails that had transparency on them like the Kado one.

Reply:
👍